compatibility/bazel_tools/testing.bzl

```
@@ -88,9 +88,6 @@ before_removing_legacy_error_codes = "2.0.0-snapshot.20220127.9042.0.4038d0a7"
 after_removing_legacy_error_codes = "2.0.0-snapshot.20220127.9042.0.4038d0a7.1"
 first_canton_in_ledger_api_tests = "2.7.0-snapshot.20230504.11748.0.af51d660"
```

In order to move coverage to canton what was covered by x before, this line should change to
```suggestion
first_canton_in_ledger_api_tests = "2.5.0"
```